src/scales/scale.radialLinear.js

```diff
@@ -321,10 +321,12 @@ export default class RadialLinearScale extends LinearScaleBase {
     LinearScaleBase.prototype.generateTickLabels.call(this, ticks);
 
     // Point labels
-    this._pointLabels = this.getLabels().map((value, index) => {
-      const label = callCallback(this.options.pointLabels.callback, [value, index], this);
-      return label || label === 0 ? label : '';
-    });
+    this._pointLabels = this.getLabels()
+      .filter((v, i) => this.chart.getDataVisibility(i))
+      .map((value, index) => {
+        const label = callCallback(this.options.pointLabels.callback, [value, index], this);
```

Comment:
The index could be considered to be wrong here, what do you thing @etimberg ?
Could just turn it around, generating all the labels and filtering afterwards.